app/drivers/kscan/kscan_gpio_matrix.c

```diff
@@ -1,48 +1,158 @@
 /*
- * Copyright (c) 2020 The ZMK Contributors
+ * Copyright (c) 2020-2021 The ZMK Contributors
```

Comment:
How do we want to handle copyright dates? Should I leave this as the original author date, or use a date range like this?